src/examples/install-cli.yml

```diff
@@ -1,10 +1,16 @@
 description: >
-  This example shows how to install the lacework-cli
+  This example shows how to install the lacework-cli.
+
+  For more information about the Lacework CLI visit:
+    - https://github.com/lacework/go-sdk/wiki/CLI-Documentation
 usage:
   version: 2.1
   orbs:
     lacework: lacework/lacework@1.2.3
-  workflows:
-    prepare_my_app:
-      jobs:
+  jobs:
+    list-events-test:
+      executor: orb-tools/ubuntu
```

Comment:
We would rather not showcase orb/tools within the examples. This example also assume the user has the orb-tools orb imported. Could we replace with a docker executor showcasing an image that would not have your CLI installed? 

And I apologize I see the example I emailed you was also incorrect.

My recommendation:


docker:
    - image: cimg/node:lts


This would replace line 12.

Reply:
example: https://github.com/CircleCI-Public/node-orb/blob/master/src/examples/install_nodejs.yml#L10

Reply:
updating!